forge/db/views/TeamType.js

```diff
@@ -0,0 +1,22 @@
+module.exports = {
+    teamType: function (app, teamType) {
+        const properties = { ...teamType.properties }
+        if (app.config.billing.stripe.teams?.[teamType.name]) {
```

Comment:
```suggestion
        if (app.config.billing?.stripe.teams?.[teamType.name]) {
```

Need to check if billing is enabled at all